docs/1-qms/13485-mapping.md

```diff
@@ -9,39 +19,39 @@ This table maps all requirements of the ISO 13485:2016 (by section) to the relev
-| 7.5     | Production and Service Provision                  | *SOP Integrated Software Development*                                                                            |
-| 7.5.5   | Special Requirements for Sterile Medical Devices  | *\- not applicable \-*                                                                                           |
-| 7.5.9   | Traceability                                      | *SOP Product Certification and Registration*                                                                     |
-| 7.6     | Control of Surveillance and Measurement           | *SOP Post-Market Surveillance*<br>*SOP Computerized Software Validation*                                         |
-| 8.1     | General Measurement, Analysis and Improvement     | *SOP Integrated Software Development*<br>*SOP Internal Auditing*<br>*SOP Management Review*                      |
-| 8.2.1   | Feedback                                          | *SOP Feedback Management*                                                                                        |
-| 8.2.2   | Complaint Processing                              | *SOP Feedback Management*<br>*SOP Corrective and Preventive Actions*                                             |
-| 8.2.3   | Reporting to Authorities                          | *SOP Incident Reporting*                                                                                         |
-| 8.2.4   | Internal Audit                                    | *SOP Internal Auditing*                                                                                          |
-| 8.2.5   | Surveillance and Measurement of Processes         | *SOP Management Review*                                                                                          |
-| 8.2.6   | Surveillance and Measurement of Products          | *SOP Post-Market Surveillance*                                                                                   |
-| 8.3     | Control of Nonconforming Products                 | *SOP Corrective and Preventive Actions*<br>*SOP Incident Reporting*                                              |
-| 8.4     | 8.4 Data Analysis                                 | *SOP Management Review*                                                                                          |
-| 8.5     | 8.5 Improvement: Corrective and Preventive Action | *SOP Corrective and Preventive Actions*                                                                          |
+| ------- | ------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------- |
+| 4.1     | General QMS Requirements                          | _Quality Management Manual_<br>_SOP Management Review_<br>_SOP Purchasing_<br>_SOP Software Validation_          |
```

Comment:
Making even small unique changes like this in a Single Change Multiple Files makes reviewing more challenging.  A quick overview is possible with SCMF, but once other changes show up, each file has to be inspected carefully...

Reply:
Yeah, it was a bit annoying that my auto-formatter ran. In the future, I'll auto-format everything first, separately, and then make the other bulk change in a separate PR.

Reply:
And just to confirm, this was the autoformatter, not a manual change.